test/integration/scheduler_perf/config/performance-config.yaml

```diff
@@ -277,6 +277,33 @@
       initPods: 5000
       measurePods: 1000
 
+# This test case simulates the scheduling of daemonset.
+# https://github.com/kubernetes/kubernetes/issues/124709
+- name: SchedulingDaemonset
+  defaultPodTemplatePath: config/templates/daemonset-pod.yaml
+  workloadTemplate:
+    # Create one node with a specific name (scheduler-perf-node),
+    # which is supposed to get all Pods created in this test case.
+  - opcode: createNodes
+    count: 1
```

Comment:
This could hide performance problems related to processing more nodes than needed.

Ideally the number of nodes should be around 15k. And number of Pods around 60k.

Reply:
> This could hide performance problems related to processing more nodes than needed.

Can you elaborate? I couldn't imagine the case.
(In next `createNodes` step, we create 15k nodes.)

> And number of Pods around 60k.

My machine couldn't handle 60k pods (timeout). 
Also, I don't see the need to increase the number of pods as each pod goes thru the exactly same scheduling process.
So, the number doesn't impacts the throughput, but only impacts the reliability of the result; and I believe 30k pods give enough for that point.